### DISCUSSION TITLE
Mailing Lists - creation and automation

### DISCUSSION THREAD

**Discussion by cortexrd** [cortexrd/Knack-Toolkit-Library#175] [Ideas]
Hi guys,

I am designing a structure to support **Mailing Lists** in Knack, along with the KTL automation power.

Here's what I have so far, but not sure if it matches what most people are looking for.

1. Shall be able to create Mailing List Context (or Category?)   Ex: Announcement, News, Discount, Marketing, Support, Investors
2. Shall be able to use existing Knack's built-in Accounts
3. Shall be able to use any other table that has an email field, Ex: Customers, Suppliers
4. Shall be able to create a new Posting with Context (above), Email Subject and Body
5. Shall be able to prepare first then schedule when to send out (should this be optional or mandatory?)
6. Shall be able to automate the process until all emails are sent
7. Shall be able to keep a log of all Postings

Any other ideas?

Norm

**Comment by amandajane-mo** [comment]
Amazing!

We have a use case where client keyworkers and managers are sent emails if
out of hours comms are created by managers.

One member of support staff is an out of hours manager after 4:30pm mon -
fri, outside of which she is classed as a support worker. Therefore, only
comms created by this user after 4:30pm should generate an email. Currently
we can have it so all her comms are emailed (leading to overload on the
mangers emails) or not sending any of her comms leading to important info
being missed by managers & keyworkers. This is probably a niche set of
circumstances and we haven’t found an elegant solution yet.

Sounds like a great addition to ktl, support for mailing list being limited
in Knack. We currently use 2 integrations to get our bulk emails sent out.

Look forward to seeing what you come up with.

Amanda
